src/EnergyPlus/HybridEvapCoolingModel.cc

```diff
@@ -0,0 +1,1591 @@
+					{
+						QTotZoneOut = 0;
+						QSensZoneOut = 0;
+						QLatentZoneOut = 0;
+						QTotSystemOut = 0;
+						QSensSystemOut = 0;
+						QLatentSystemOut = 0;
+
+					}
+					//set cooling specific outputs
+					if (CoolingRequested )
+					{
+						UnitTotalCoolingRate = std::abs( QTotZoneOut);
+						UnitTotalCoolingEnergy = UnitTotalCoolingRate * TimeStepSys * SecInHour;
+						UnitSensibleCoolingRate = std::abs( QSensZoneOut );
+						UnitSensibleCoolingEnergy = UnitSensibleCoolingRate * TimeStepSys * SecInHour;
+						UnitLatentCoolingRate = UnitTotalCoolingRate - UnitSensibleCoolingRate;
+						UnitLatentCoolingEnergy = UnitTotalCoolingEnergy - UnitSensibleCoolingEnergy;
+
+						SystemTotalCoolingRate = std::abs( QTotSystemOut);
+						SystemTotalCoolingEnergy = SystemTotalCoolingRate * TimeStepSys * SecInHour;
+						SystemSensibleCoolingRate = std::abs( QSensSystemOut );
+						SystemSensibleCoolingEnergy = SystemSensibleCoolingRate * TimeStepSys * SecInHour;
+						SystemLatentCoolingRate = SystemTotalCoolingRate - SystemSensibleCoolingRate;
+						SystemLatentCoolingEnergy = SystemTotalCoolingEnergy - SystemSensibleCoolingEnergy;
+					}
+					else
+					{
+						UnitTotalCoolingRate = UnitTotalCoolingEnergy = UnitSensibleCoolingRate = UnitSensibleCoolingEnergy= UnitLatentCoolingRate= UnitLatentCoolingEnergy = 0;
+						SystemTotalCoolingRate = SystemTotalCoolingEnergy = SystemSensibleCoolingRate = SystemSensibleCoolingEnergy = SystemLatentCoolingRate = SystemLatentCoolingEnergy = 0;
+
+					}
+					// set heating specific outputs
+					if (HeatingRequested)
+					{
+						UnitTotalHeatingRate = std::abs(QTotZoneOut);
+						UnitTotalHeatingEnergy = UnitTotalHeatingRate * TimeStepSys * SecInHour;
+						UnitSensibleHeatingRate = std::abs( QSensZoneOut );
+						UnitSensibleHeatingEnergy = UnitSensibleHeatingRate * TimeStepSys * SecInHour;
+						UnitLatentHeatingRate = UnitTotalHeatingRate - UnitSensibleHeatingRate;
+						UnitLatentHeatingEnergy = UnitTotalHeatingEnergy - UnitSensibleHeatingEnergy;
+
+						SystemTotalHeatingRate = std::abs( QTotSystemOut);
+						SystemTotalHeatingEnergy = SystemTotalHeatingRate * TimeStepSys * SecInHour;
+						SystemSensibleHeatingRate = std::abs( QSensSystemOut);
+						SystemSensibleHeatingEnergy = SystemSensibleHeatingRate * TimeStepSys * SecInHour;
+						SystemLatentHeatingRate = SystemTotalHeatingRate - SystemSensibleHeatingRate;
+						SystemLatentHeatingEnergy = SystemTotalHeatingEnergy - SystemSensibleHeatingEnergy;
+					}
+					else
+					{
+						UnitTotalHeatingRate = UnitTotalHeatingEnergy = UnitSensibleHeatingRate = UnitSensibleHeatingEnergy = UnitLatentHeatingRate = UnitLatentHeatingEnergy = 0;
+						SystemTotalHeatingRate = SystemTotalHeatingEnergy = SystemSensibleHeatingRate = SystemSensibleHeatingEnergy = SystemLatentHeatingRate = SystemLatentHeatingEnergy = 0;
+
+					}
+				FinalElectricalPower = 1000 * CalculateTimeStepAverage(SYSTEMOUTPUTS::SYSTEM_FUEL_USE);
```

Comment:
Is it clear that the calculations above are, apparently, in a different set of units, such that you have to multiply by 1000 here?  I never saw it above, and this is a scary bug-maintenance concern.